sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentPage.java

```diff
@@ -145,7 +145,7 @@ private void setHeight(Float height) {
      *
      * @return the unit value.
      */
-    public LengthUnit getUnit() {
+    public DocumentPageLengthUnit getUnit() {
```

Comment:
Just curious why these got DocumentPage appended to them?

Reply:
It is what @kinelski mentioned here in his PR:

>To avoid models with the same name across the old and new Form Recognizer namespaces, CopyAuthorization, LengthUnit, and SelectionMarkState have been renamed to DocumentModelCopyAuthorization, DocumentPageLengthUnit, and DocumentSelectionMarkState, respectivelly.